index.html

```diff
@@ -57,7 +57,7 @@ <h3>Project title 1</h3>
 		
 		</section>
 	
-		<section class= "about">  <!-- section contating a breif bio and an image of me -->
```

Comment:
These comments are useful but generally not used in the industry so best to remove them (I had the same and that's what Rayna said to me :) ).